src/django_upgrade/fixers/request_headers.py

```diff
@@ -58,6 +58,44 @@ def visit_Call(
         )
 
 
+@fixer.register(ast.In)
+def visit_In(
+    state: State,
+    node: ast.In,
+    parents: list[ast.AST],
+) -> Iterable[tuple[Offset, TokenFunc]]:
+    if (
+        type(parents[-1]) is ast.Compare
```

Comment:
it's nearly always better to use `isinstance()` for type checking - it allows subclasses

```suggestion
        isinstance(parents[-1], ast.Compare)
```